bin/normalize-ways

```diff
@@ -74,10 +76,16 @@ waysStream.on('data', function (line) {
     for (var i = 0; i < ways.length - 1; i++) {
       var way = ways[i];
       way.properties.id = way.properties.id + ';' + quadkey;
+      if (openLR) {
+        way.properties.openLR = geojsonLR.toOpenLRString(way);
+      }
       outputStream.write(JSON.stringify(way) + '\n');
     }
     way = ways[ways.length - 1];
     way.properties.id = way.properties.id + ';' + quadkey;
+    if (openLR) {
+      way.properties.openLR = geojsonLR.toOpenLRString(way);
```

Comment:
do you think there is a way to avoid having to do this line (and the one before) twice?

Reply:
Lines 84-90 are a strange way of adding the last line.  I know it was not part of this PR, but it'd be awesome if there is a way of avoiding it. All good if you think this is OK for now.

Reply:
yes I agree there is probably a more elegant pattern here but let's save it for a different PR